### PR TITLE
Add address to impressum

### PR DESCRIPTION
**Pull request by edolstra**

### CODE REVIEW

File: impressum.md [hunk 1]
```diff
@@ -10,11 +10,9 @@
...
+**Address:** Korte Lijnbaanssteeg 1-4318, 1012 SL, Amsterdam, Netherlands
+
 ## Bank details
 
 * IBAN: NL13 BUNQ 2061 2530 32
 * BIC:  BUNQNL2AXXX
-
-## Managing Directors
-
-Eelco Dolstra
```
Comment (mweinelt) [inline impressum.md:20]: Seems unrelated to the change described in the commit message.

Comment (edolstra) [reply impressum.md:20]: It is, but I cleaned this up while I was at it. (I don't know why I'm there as a "Managing Director".)

Comment (fricklerhandwerk) [reply impressum.md:20]: This could have been because of pattern matching required by some entity the foundation cooperated with.
